config/cobbler/distro_signatures.json

```diff
@@ -1336,32 +1305,6 @@
         "kernel_options": "",
         "kernel_options_post": "",
         "boot_files": []
-      },
-      "focal": {
```

Comment:
Why remove ubuntu focal?

Reply:
Fixed. Thanks for the review. Copy Pasta Error...